backend/seeds/00800_matches.js

```diff
@@ -0,0 +1,25 @@
+const { tableStatusUpdates } = require('../src/models/statusUpdate')
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex(tableStatusUpdates)
+    .del()
+    .then(function () {
+      // Inserts seed entries
+      return knex(tableStatusUpdates).insert([
+        {
+          StudentID: knex('students').where({
+            FullName: 'fullname',
+            PhoneNumber: '12345678' 
+          }).select('StudentID'),
+          PreviousStatusID: knex('statuses').where({
+            Description: 'UNMATCHED' 
+          }).select('StatusID'),        
+          NextStatusID: knex('statuses').where({
+            Description: 'SCREENING' 
+          }).select('StatusID'), 
```

Comment:
I think we don't need to store Previous/Next statuses.

In the Jira ticket, we agreed to store the ID of the student's Status _Update_ (from the `statusUpdates` table). You have reflected this in the Match model below, where it's implied that there is a `StudentStatusUpdateID` column).

Reply:
(this file has since been deleted)